client/state/sharing/publicize/reducer.js

```diff
@@ -37,9 +43,11 @@ export function fetchingConnections( state = {}, action ) {
 		case SERIALIZE:
 			return state;
 		case DESERIALIZE:
-			return state;
+			if ( isValidStateWithSchema( state, fetchingConnectionsSchema ) ) {
```

Comment:
I wonder how intentional we should be here about intentionally excluding items from state persistence. For example, fetching status does not seem like something that should be persisted, since it's applicable only in the context of the current browser session.

There's a potentially related discussion off of this in regards to whether fetching status makes sense to keep in the sharing tree (an otherwise "entity" tree) vs. a more disposable tree segment e.g. `ui` (related: #3337).